.gitignore

```diff
@@ -205,7 +206,8 @@ $RECYCLE.BIN/
 .LSOverride
 
 # Icon must end with two \r
-Icon


```

Comment:
This is actually important. Can you revert this?

Reply:
Try `git checkout main .gitignore`

Reply:
Totally forgot you could do this, thanks!

reverted in https://github.com/seamapi/react/pull/47/commits/ec209b813c7886dddcb391d4a8ac9777ff7acb9c